default/scripting/techs/defense/mines.macros

```diff
@@ -1,6 +1,7 @@
 // EffectsGroups for System Defense Mines
 // arg1 total damage
 // arg2 priority
+// arg3 planet scope (-1=all systems with an empire planet, 1=source system only)
```

Comment:
I question whether this is a good use of macro parameters, both for readability and evaluation efficiency reasons...